src/Dialect/ONNX/ONNXOps/Math/ElementwiseBroadcast.cpp

```diff
@@ -343,14 +343,7 @@ LogicalResult ONNXOrOp::inferShapes(
 //===----------------------------------------------------------------------===//
 
 LogicalResult ONNXPowOp::verify() {
```

Comment:
do we need an explicit verifier anymore? or do all onnx ops opt into `hasVerifier`? (i dont remember)